website/other/bezier-rs-demos/src/features/bezier-features.ts

```diff
@@ -483,6 +483,54 @@ const bezierFeatures = {
 		},
 		chooseTVariant: true,
 	},
+	"join": {
+		name: "Join",
+		callback: (bezier: WasmBezierInstance): string => {
+			const points = JSON.parse(bezier.get_points());
+			if (points.length === 2) {
+				return bezier.join([
+					[120, 155],
+					[40, 155],
+				]);
+			}
+			if (points.length === 3) {
+				return bezier.join([
+					[40, 150],
+					[95, 195],
+					[155, 145],
+				]);
+			}
+			return bezier.join([
+				[140, 150],
+				[85, 110],
+				[65, 180],
+				[30, 140],
+			]);
```

Comment:
It'd be a bit more idiomatic to use `if-elseif-else` to avoid needing to use the explicit `return` keyword.